infra.bs

```diff
@@ -74,13 +72,14 @@ marked non-normative. Everything else is normative.
 
 <p>The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
 "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in
-RFC 2119. [[!RFC2119]]
+<cite>RFC 2119</cite>. [[!RFC2119]]
```

Comment:
I tend to prefer citing the actual titles, in this case "Key words for use in RFCs to Indicate Requirement Levels". I'm not sure RFC 2119 is actually the title of a work, so if you leave it in that form, I think omitting the `<cite>` is best.